### ISSUE TITLE
Url passed to transformUrl should be normalized

### ISSUE THREAD

**Issue by adamreisnz** [miickel/gulp-angular-templatecache#115]
On OS X, the url passed to the transformUrl function is normal, e.g. something like `components/app/app.html`. However, that same url on Windows becomes `components//app//app.html`. 

It seems to be because somewhere along the line the Windows backslashes are converted to forward slashes, and for some reason an additional slash is added. This means you have to take into account when writing the transformUrl function, that the input you get is not going to be consistent across different OS's.

Is it possible to let the gulp-angular-templatecache normalize the url before passing it to the transformUrl function, so that there are no OS dependent differences?

**Comment by hungdoan2** [comment]
:+1: 
temporary solution is  use `path.normalize` to normalize your url before `return` in `transformUrl `

**Comment by simonua** [comment]
@adamreisnz, @hung-doan, is this still relevant? If so, could you create a PR, please? If not, could you close the issue, please?

**Comment by adamreisnz** [comment]
This is still an issue AFAIK yes, but I'm not very interested in development for windows, just wanted to report the issue.

**Comment by adamreisnz** [comment]
It's still an issue AFAIK, and just because the community is not interested in submitting a PR at this stage (I don't have access to windows computers in the first place), doesn't mean there hasn't been response.

**Comment by simonua** [comment]
That's fair, and this can be viewed from different perspectives, but in order to do housekeeping on open issues, it's not uncommon to close issues that have remained open for a long time without any progress. I should mention again that I'm not opposed to reopening it at all.

**Comment by adamreisnz** [comment]
True, but I think it'd be more productive to label it as "help wanted" and leave it open, so that if anyone does feel so inclined, they can pick up these issues. No one is going to sift through the list of closed issues to look for things to do, so essentially this issue is just being swept under the rug ;)

**Comment by simonua** [comment]
@adamreisnz, good conversation. Let's go with a "help wanted" issue then. Thank you.

**Comment by adamreisnz** [comment]
Awesome, thanks for that!

**Comment by simonua** [comment]
Yeah, you bet. Good arguments in a productive conversation. Thanks for chiming in, Adam!
